wiki/People/The_Team/id.md

```diff
@@ -1,56 +1,84 @@
----
-outdated: true
----
-
 # Tim
 
-## Pelopor pembuatan
+*Untuk daftar pengguna yang dipromosikan atau telah pensiun, lihat: [Staff Log](/wiki/Staff_Log)*
 
-| Nama | Deskripsi |
-| ---- | ----------- |
-| [peppy](https://osu.ppy.sh/users/2) | Pemimpin proyek |
+Para pengguna yang tercantum di bawah adalah **tim inti osu!** dan bertanggung jawab untuk membantu menjadikan permainan ini luar biasa bagi semua orang.
 
-## Aktif
+| Nama | Peran <!-- TODO: "Featured Artist outreach" isn't a role but I'm not sure how to write it as one... --> |
+| :-- | :-- |
+| ![][flag_AU] **[peppy](https://osu.ppy.sh/users/2)** | **Pemimpin proyek** |
+| ![][flag_GB] [-Mo-](https://osu.ppy.sh/users/2202163) | Penasihat adegan *modding* dan *mapping*|
```

Comment:
Is there a reason for why "modding" and "mapping" are italicized? I tried looking for the diff and could not find if this was how it was originally in ID or EN.